flashlib.core.robot/src/main/java/com/flash3388/flashlib/robot/motion/supervised/SupervisedMovable.java

```diff
@@ -0,0 +1,54 @@
+package com.flash3388.flashlib.robot.motion.supervised;
+
+import com.flash3388.flashlib.control.Direction;
+import com.flash3388.flashlib.control.Stoppable;
+import com.flash3388.flashlib.scheduling.Requirement;
+
+public interface SupervisedMovable extends Stoppable, Supervisable, Requirement {
```

Comment:
You could just extend `Movable`... You wouldn't need to provide the defaults in that case.

Do the same with the rest.